doc/sphinx/conf.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# osgar documentation build configuration file, created by
+# sphinx-quickstart on Sun Oct 28 10:25:10 2018.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.githubpages']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'osgar'
+copyright = '2018, robotika.cz'
+author = 'robotika.cz'
```

Comment:
Please try if one of these can be a link. I would be nice if the bottom copyright notice was a link to robotika.cz

Reply:
well, yes, it would be nice, but ... so far the closest info I found here:
https://groups.google.com/forum/#!topic/sphinx-users/NGfqBDiNo5c
(unresolved)
... it is maybe then easier to generate pages/footer ourselves?

Reply:
I thought it would be easy. If it is not, let's skip it for now.